charts/drupal/templates/backup-cron.yaml

```diff
@@ -28,17 +34,38 @@ spec:
             command: ["/bin/bash", "-c"]
             args:
               - |
-                {{ include "drupal.backup-command" . | nindent 16 }}
+                {{ include "drupal.backup-command" (merge $params . ) | nindent 16 }}
+                {{ include "mariadb.db-validation" (merge $params . ) | nindent 16 }}
+            securityContext:
+              capabilities:
+                #For shareProcessNamespace
+                add:
+                  - SYS_PTRACE
+          - name: mariadb
+            image: docker.io/bitnami/mariadb:10.2.32
```

Comment:
Is it possible to skip the patch version on the image tag?

Reply:
Yes, mariadb image repo provides a tag  with major.minor version. 
Updated in next commit.